### PR TITLE
Implement Squad Maker

### PR DESCRIPTION
**Pull request by Benobi42**
A new pull request due to major changes as a result of the previous review.

A working demo is available at http://206.189.222.135

### CODE REVIEW

File: players.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+# Default JSON Data file
+JSONFILE = "players.json"
+
+SKILLTYPES = ["Skating", "Shooting", "Checking"]
+
+
+def getSkillRating(sklData, sklType):
+    """
+    Get the rating for a given skill from
+    a single player's skill data
```
Comment (eklinger-UofA) [inline players.py:14]: Here and in a few places there are sentences that don't end in a '.'

File: players.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+from flask_table import Table, Col
+
+# Default JSON Data file
+JSONFILE = "players.json"
+
+SKILLTYPES = ["Skating", "Shooting", "Checking"]
+
+
+def getSkillRating(sklData, sklType):
```
Comment (eklinger-UofA) [inline players.py:11]: The skl abbreviation only saves you 2 characters and makes someone else think a second about what it acutally means. Suggest just going with 'skill'

File: players.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    return [skl for skl in sklData if skl['type'] == sklType][0]['rating']
+
+
+class Player:
+    """Representation of a Hockey Player
+
+    A Player contains a unique identifier, a name, and a series
+    of skill ratings that correspond to the skill types at the
+    head of the file.
+
```
Comment (eklinger-UofA) [inline players.py:30]: Extra space here, most of your class/method comments don't have this

File: players.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    """
+    def __init__(self, id, name,
+                 skateRating, shootRating, checkRating):
```
Comment (eklinger-UofA) [inline players.py:33]: Line break here feels awkward having the second line being longer than the first. up to you and your style

File: players.py [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+            data = json.load(f)
+        for player in data["players"]:
+            p = Player.fromJSON(player)
+            players.append(p)
+        return cls(players)
+
+
+class PlayerTable(Table):
+    """
+
```
Comment (eklinger-UofA) [inline players.py:131]: Blank line to start?

**Review by eklinger-UofA — APPROVED** [review]
A few things, all cosmetic

**Comment by Benobi42** [comment]
Closing due to major changes, will reopen

**Pull request closed without merging**